systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -153,6 +157,8 @@
     private static final ClusterOperatorInstallType CLUSTER_OPERATOR_INSTALL_TYPE_DEFAULT = ClusterOperatorInstallType.BUNDLE;
     private static final boolean LB_FINALIZERS_DEFAULT = false;
     private static final String STRIMZI_FEATURE_GATES_DEFAULT = "";
+
+    private static final String CLIENTS_KAFKA_VERSION_DEFAULT = "3.0.0";
```

Comment:
Maybe you can move `ST_KAFKA_VERSION` next to it and also add prefix `ST_` ? Or at least move it

Reply:
Okay, I wanted test-clients relative defaults to be together. But I'll move it and rename it, thanks :)